sdkproject/Assets/Mapbox/Unity/Editor/MapManagerEditor.cs

```diff
@@ -158,6 +181,36 @@ public override void OnInspectorGUI()
 
 			ShowSepartor();
 
+			ShowLocationPrefabs = EditorGUILayout.Foldout(ShowLocationPrefabs, "LOCATION PREFABS");
+			if (ShowLocationPrefabs)
+			{
+				var vectorDataProperty = serializedObject.FindProperty("_vectorData");
+
+
+				var layerProperty = vectorDataProperty.FindPropertyRelative("_layerProperty");
+				var layerSourceProperty = layerProperty.FindPropertyRelative("sourceOptions");
+				var sourceType = layerProperty.FindPropertyRelative("_sourceType");
+				VectorSourceType sourceTypeValue = (VectorSourceType)sourceType.enumValueIndex;
+				string streets_v7 = MapboxDefaultVector.GetParameters(VectorSourceType.MapboxStreets).Id;
+				//var layerSourceId = vectorDataProperty.FindPropertyRelative("Id");
```

Comment:
Should we just remove some of these commented-out lines?

Reply:
Agree with @morgane - Remove commented out lines.